cli/main.c

```diff
@@ -940,23 +962,19 @@ static int port_bind_info(int argc, char **argv)
 
 	ret = switchtec_bind_info(cfg.dev, &bind_status, cfg.phy_port);
 
-	if (ret < 0) {
-		printf("bind_info_error: %f", ret);
+	if (ret > 0) {
```

Comment:
You still need to handle the case where switchtec_bind_info() returns less than zero (indicating an error submitting the command or something like that.

Reply:
The function switchtec_bind_info()/switchtec_bind()/switchtec_unbind() cannot return a value that is less then zero. If the return value is bigger than zero, it indicates a error.

Reply:
That's wrong. switchtec_bind_info() calls switchtec_cmd() and returns the result. switchtec_cmd() can always return negative numbers.